src/charts/grid.tsx

```diff
@@ -180,7 +180,7 @@ class DataResourceTransformGrid extends React.PureComponent<Props, State> {
       ) {
         return {
           Header: field.name,
-          accessor: field.name,
+          accessor: (rowValue: { [key: string]: any }) => rowValue[field.name],
```

Comment:
This makes sense to me. Do we need to do the same bool => string mapping done in the check below?

https://github.com/nteract/data-explorer/blob/20e75b839c64b205ab4f3630a48a4338e883b412/src/charts/grid.tsx#L207-L208

Reply:
Good question: I think it's unnecessary since we do have confidence that the `field.type` can't be boolean based on this check:

https://github.com/nteract/data-explorer/blob/20e75b839c64b205ab4f3630a48a4338e883b412/src/charts/grid.tsx#L176-L180

It is possible that the schema could be malformed and that there could be booleans inside `rowValue[field.name]` if the column naming is inaccurate, but that's a different issue.